packages/fx-core/src/common/projectSettingsHelper.ts

```diff
@@ -115,7 +118,11 @@ export function isPureExistingApp(projectSettings: ProjectSettings): boolean {
 }
 
 export function getProjectSettingsVersion() {
-  return "2.1.0";
+  if (isBotNotificationEnabled()) {
+    return "2.2.0";
```

Comment:
what's this for?

Reply:
A new enum notifiation for "capabilites" property is added. Should we update to projectSettings version?

```
"azureResources“: [],
"capabilities": [
  "Notification",
]
```

Reply:
Is `capabilities` related to teams app manifest? If so, just reuse the `bot` value.
IMO, no need to distinguish notification/command bot since all the differences are in code template.

Reply:
Fixed. Now, no config file schema is changed. Use `answers["scenario"]` and `PluginActRoles` to distinguish notification bot and command bot